pkg/capabilities/consensus/ocr3/types/aggregator.go

```diff
@@ -10,14 +10,15 @@ import (
 const MetadataFieldName = "INTERNAL_METADATA"
 
 type Metadata struct {
-	Version       uint32 // 1 byte
-	ExecutionID   string // 32 hex bytes
-	Timestamp     uint32 //  4 bytes
-	DONID         string //  4 hex bytes
-	WorkflowID    string // 32 hex bytes
-	WorkflowName  string // 10 hex bytes
-	WorkflowOwner string // 20 hex bytes
-	ReportID      string //  2 hex bytes
+	Version          uint32 //  1 byte
+	ExecutionID      string // 32 hex bytes
+	Timestamp        uint32 //  4 bytes
+	DONID            string //  4 hex bytes
+	DONConfigVersion uint32 //  4 bytes
```

Comment:
@krehermann added this field since you last reviewed. It's set to zero now anyway, until Syncer is ready.

Reply:
Are we not passing it from the workflow for now, like the donID? They should be sourced from the same place in the future

Reply:
Yeah it will be passed in the same way but we won't have it available until the Syncer is ready.

Reply:
hmm do you mean that the logic to actually pass it though all the layers is missing... you're right, let me add it